client/cmd/internal/plan/operation.go

```diff
@@ -0,0 +1,40 @@
+package plan
+
+type Operation int
+
+const (
+	OperationDelete Operation = iota + 1
+	OperationCreate
+	OperationUpdate
+)
+
+func (o Operation) String() string {
```

Comment:
how are we handling migrate

Reply:
we will handling migrate using plan.Compositor to merge migrate operation (change namespace and update on apply command) between create and delete operation